CliWrap/Exceptions/CommandExecutionException.cs

```diff
@@ -7,48 +7,57 @@ namespace CliWrap.Exceptions
     /// </summary>
     public partial class CommandExecutionException : Exception
     {
+        public ICommandConfiguration Command { get; }
+        public int ExitCode { get; }
+        public string? StandardError { get; }
```

Comment:
I don't think we should expose this here. If needed, I'd rather we make a derived exception, i.e. `BufferedCommandExecutionException`, but for now I think it's best not to include this.